spaces/domains.json

```
@@ -15,6 +15,7 @@
   "vote-celo.goodghosting.com": "celo.goodghosting.eth",
   "vote-polygon.goodghosting.com": "polygon.goodghosting.eth",
   "vote.the-figure.com": "thefiguretoken.eth",
+  "everydaydao": "andrewladdusaw.eth",
```

Your domain is missing an extension.
```suggestion
  "everydaydao.com": "andrewladdusaw.eth",
```

---

Can you make it "everydaydao.xyz"?